domaintools_iris_connector.py

```diff
@@ -54,7 +55,7 @@ def initialize(self):
         self.app_version_number = app_json_configuration.get("app_version", "")
         self.app_name = app_json_configuration.get("name", "")
         self.app_partner = "splunk_soar"
-        self._rest_url = self._build_rest_url()
+        self._rest_url = phrules.build_phantom_rest_url()
```

Comment:
This can use `BaseConnector.get_phantom_base_url` and avoid importing `phantom.rules`

```suggestion
        self._rest_url = f"{self.get_phantom_base_url()}/rest"
```